buttercup/cogs/find.py

```diff
@@ -187,40 +216,43 @@ def get_submission_from_url(self, reddit_url: str) -> Union[Dict, None]:
     )
     async def _find(self, ctx: SlashContext, reddit_url: str) -> None:
         """Find the post with the given URL."""
-        self._clear()
         if (reddit_url := normalize_url(reddit_url)) is None:
             await ctx.send(
                 "Sorry, that doesn't look like a URL I can work with."
```

Comment:
Can we add the URL to the response? I think you can't see your original command anymore, so you can't "troubleshoot" why the URL is invalid

Reply:
![URL not being shown when it is incorrect](https://user-images.githubusercontent.com/13908946/122189827-8ca2ee80-ce91-11eb-9c7f-471d4da84dd1.png)

Reply:
Great point!

Reply:
Alright I fixed this.